site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc

```diff
@@ -0,0 +1,45 @@
+[api]
+ifeval::["{base_field_name}" != "identityProviderLink"]
+[field]#identityProviderLinks# [type]#[Array]#::
+The list of Link objects.
+endif::[]
+
+[field]#{base_field_name}.displayName# [type]#[String]#::
+The display name for the user.
```

Comment:
This field is really just here to give a human readable name to the link. For example, if you were to have two links to Twitter, this will help differentiate. 

Each IdP implementation decides what to put here, it will be a handle, username or email depending upon the IdP. 

This may be best described as 

> A human readable name for this link to help you identify this link. This value will generally be an email address, or username. This value is recorded when the link is made, and updated during the next login for this identity provider. This value may not necessarily reflect the username or email you used to authenticate with the 3rd party identity provider.